File: app/src/main/java/com/androdevlinux/percy/stackingsats/pojo/payment/PaymentMethodInstructionsResponseBean.java
package com.androdevlinux.percy.stackingsats.pojo.payment;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class PaymentMethodInstructionsResponseBean {

    @SerializedName("status")
    @Expose
    private String status;
    @SerializedName("payment_method_instruction")
    @Expose
    private PaymentMethodInstruction paymentMethodInstruction;

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public PaymentMethodInstruction getPaymentMethodInstruction() {
        return paymentMethodInstruction;
    }

    public void setPaymentMethodInstruction(PaymentMethodInstruction paymentMethodInstruction) {
        this.paymentMethodInstruction = paymentMethodInstruction;
    }

}

File: app/src/main/java/com/androdevlinux/percy/stackingsats/pojo/payment/PaymentMethodInstructionsResponseBean.kt
package com.androdevlinux.percy.stackingsats.pojo.payment

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class PaymentMethodInstructionsResponseBean {
    @SerializedName("status")
    @Expose
    var status: String? = null

    @SerializedName("payment_method_instructions")
    @Expose
    var paymentMethodInstructions: List<PaymentMethodInstruction>? = null
}